docs/capacity-planning.rst

```diff
@@ -53,7 +53,7 @@ Kafka
 
 KSQL consumes resources on your Kafka cluster.
 
-Internal Topics
+Generated Topics
```

Comment:
Per RST guidelines, the header symbols match the length of the header exactly, or else it won't render. So in this case you need to add a few extra `+` below :(

Reply:
Aha! I didn't know that. Thanks :)
Added an extra `+` to match the length!

Reply:
Yea, looks good. RST is quite a PITA.